### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by incorvia** [mcufont/mcufont#11]
This seems like a great library but the documentation seems to be sparse so I'm having trouble piecing together aspects of how to use the encoder.  Some additions could be greatly appreciated.

**Comment by PetteriAimonen** [comment]
Yeah, unfortunately I've pretty much abandoned it due to lack of time.

**Comment by kerryland** [comment]
Fixed in 2021! You can close this one :-)
